src/snmp_agent.cc

```diff
@@ -592,11 +592,11 @@ snmp_prfProtoFn(variable_list * Var, snint * ErrP)
             break;
 
         case PERF_MEDIAN_RHR:
-            x = statRequestHitRatio(minutes);
+            x = statRequestHitRatio(minutes).toPercentOr(0);
```

Comment:
Do we really need to always pass the default '0' value? Or there will be in the future some non-zero cases toPercentOr(someNonZeroValue)?